src/main/java/tech/jhipster/lite/generator/client/hexagonal_documentation/infrastructure/primary/FrontHexagonalDocumentationModuleConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package tech.jhipster.lite.generator.client.hexagonal_documentation.infrastructure.primary;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+import tech.jhipster.lite.generator.client.hexagonal_documentation.application.FrontHexagonalArchitectureDocumentationApplicationService;
+import tech.jhipster.lite.generator.slug.domain.JHLiteModuleSlug;
+import tech.jhipster.lite.module.domain.resource.JHipsterModuleResource;
+
+@Component
+class FrontHexagonalDocumentationModuleConfiguration {
+
+  @Bean
+  JHipsterModuleResource frontHexagonalDocumentationModule(FrontHexagonalArchitectureDocumentationApplicationService documentation) {
+    return JHipsterModuleResource.builder()
+      .slug(JHLiteModuleSlug.FRONT_HEXAGONAL_ARCHITECTURE)
+      .withoutProperties()
+      .apiDoc("Front", "Add front hexagonal architecture documentation")
```

Comment:
```suggestion
      .apiDoc("Frontend", "Add front hexagonal architecture documentation")
```
For consistency with other modules: https://github.com/search?q=repo%3Ajhipster%2Fjhipster-lite+.apiDoc%28%22Frontend&type=code